cmd/add.go

```
@@ -0,0 +1,93 @@
+`, name, targetURL, time.Now().Format(time.RFC3339), scriptPath)
+
+		err := os.WriteFile(targetPath, []byte(targetContent), 0644)
+
+		if err != nil {
+			fmt.Println("Error writing target file:", err)
+			os.Exit(1)
+		}
+		fmt.Printf("Created target file: %s\n", targetPath)
+
+		if _, err := os.Stat(scriptPath); os.IsNotExist(err) {
+			os.WriteFile(scriptPath, []byte(scriptStub(scriptLang)), 0755)
```

golangci-lint (1.64.8)

Error return value of `os.WriteFile` is not checked

(errcheck)